planning/behavior_path_planner/config/avoidance/avoidance.param.yaml

```diff
@@ -6,6 +6,8 @@
       resample_interval_for_output: 4.0                 # [m]
       detection_area_right_expand_dist: 0.0             # [m]
       detection_area_left_expand_dist: 1.0              # [m]
+      drivable_area_right_bound_offset: 0.0             # [m]
+      drivable_area_left_bound_offset: 0.0              # [m]
```

Comment:
It already exists in the other config file.

https://github.com/autowarefoundation/autoware.universe/blob/4f0769fd3daa561b6bd3642f14a43cf3a6670290/planning/behavior_path_planner/config/drivable_area_expansion.param.yaml#L4-L7

Reply:
Thank you for checking!
But in launcher side same parameters exist. Launcher side should be deleted?

https://github.com/autowarefoundation/autoware_launch/blob/32c4dadd1254c189ccfcca205e246661014d026e/autoware_launch/config/planning/scenario_planning/lane_driving/behavior_planning/behavior_path_planner/avoidance/avoidance.param.yaml/#L9-L10

Reply:
Yes, I think we should remove both side parameters.

Reply:
I understand!
Deleted params in `avoidance.param.yaml`